### DISCUSSION TITLE
Failed Speedtest not shown in results anymore

### DISCUSSION THREAD

**Discussion by SodaSyrup** [alexjustesen/speedtest-tracker#1623] [Q&A]
Hello,
i recently had a problem with scheduled tests, as i didnt know how to set them up without the settings tab, that got resolved. this left me with a big gap of not testing, so after learning how to actually schedule it, the container was redeployed and is now running fine, but i feel like failed tests arent shown in the results anymore. either my isp fixed everything or is there a setting that needs to be set for it to not disregard failed tests?

**Comment by svenvg93** [comment]
Failed test are either marked as failed, or in some cases it stays in started state in the results table. They are not being discarded or anything.

geuss your isp fixed there problems

**Reply by dukrous** [reply]
Previously failed tests were shown on the graph as a zero result and it was easy to see outages or other issues. Today, for example, I'm in the middle of a 2 hour and growing outage and I can't see it on the tracker. 

It would be nice if failed tests showed as a zero result on the graph again.

**Comment by mmomjian** [comment]
I can confirm this, I am not sure if this is a regression but Failed tests do not appear on the graph at all. The graph just stacks the "good" tests next to each other with no apparent gap/issue.

**Comment by svenvg93** [comment]
This listed here as an FR; https://github.com/alexjustesen/speedtest-tracker/issues/509

If the failed test ever soon then it was a bug 😅 . As we explicitly filter on `Completed` results 

https://github.com/alexjustesen/speedtest-tracker/blob/37abcd1f2b6cbc323d33b17f699e0f70becde31c/app/Filament/Widgets/RecentDownloadChartWidget.php#L32-L39

**Reply by mmomjian** [reply]
Thanks. I searched but I didn’t find that one. Appreciate it
